ldpd/lde.c

```diff
@@ -1620,10 +1620,8 @@ lde_address_list_free(struct lde_nbr *ln)
 {
 	struct lde_addr		*lde_addr;
 
-	while ((lde_addr = TAILQ_FIRST(&ln->addr_list)) != NULL) {
-		TAILQ_REMOVE(&ln->addr_list, lde_addr, entry);
+	while ((lde_addr = TAILQ_POP_FIRST(&ln->addr_list, entry)) != NULL)
```

Comment:
I actually think that TAILQ should have a "POP_FIRST" macro anyway because the macro is kindof simpler;  I was rather surprised that there isn't one.  It's a frequent use case (as seen in our code) to get and remove the first item, and having this in one macro helps to avoid stupid mistakes.